keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepositoryTests.kt

```diff
@@ -327,16 +329,17 @@ abstract class CombinedRepositoryTests<D : DeliveryConfigRepository, R : Resourc
             }
 
             test("stores the updated resource") {
-              expectThat(subject.getResource(resource.id))
-                .get { spec }
+              expectThat(subject.getResource(resource.id)) {
+                get(Resource<*>::version) isEqualTo 2
+                get(Resource<*>::spec)
                 .isA<DummyResourceSpec>()
-                .get { data }
-                .isEqualTo("kthxbye")
+                .get(DummyResourceSpec::data) isEqualTo "kthxbye"
+              }
             }
 
             test("records that the resource was updated") {
               verify {
-                resourceRepository.appendHistory(ofType<ResourceCreated>())
+                resourceRepository.appendHistory(ofType<ResourceUpdated>())
               }
             }
```

Comment:
this test was wrong 😬

Reply:
Oops, my bad. How was it passing?! 😬

Reply:
Very good question. I presume because that type of event was added as well (but that was already checked in an earlier test)

Reply:
Argh. Thanks for fixing it! Damn copy/🍝.